docs/getting-started/ui/index.rst

```diff
@@ -0,0 +1,201 @@
+
+In this situation, the ``Oso.get_allowed_actions()`` method can be very
+helpful. The method returns a list of actions that a user is allowed to
+take on a specific resource. In our example, we call
+``get_allowed_actions()`` in the route handler for the "Show repository"
+view to get the user's allowed actions for the current repo:
+
+.. code:: python
+
+    def repos_show(org_id, repo_id):
+        # Get repo
+        repo = Repository.query.get(repo_id)
+        
+        # Authorize repo access
+        current_app.oso.authorize(repo, action="READ")
+
+        # Get allowed actions on the repo
+        actions = current_app.base_oso.get_allowed_actions(
+            get_current_user(), repo
+        )
+            # Send allowed actions to template (or frontend)
+        return render_template(
+            "repos/show.html",
+            repo=repo,
+            org_id=org_id,
+            actions=actions,
+        )
+
+In our demo app, when we call ``get_allowed_actions`` with the user
+``mike@monsters.com``, we get back:
+
+.. code:: python
+
+    actions = ['READ', 'LIST_ROLES', 'CREATE', 'DELETE', 'LIST_ISSUES']
+
+But when we call with a different user, ``sully@monsters.com``, we get:
+
+.. code:: python
+
+    actions = ['READ', 'LIST_ISSUES', 'CREATE']
+
+The allowed actions for each user are determined by the **oso policy.**
+In this case, our policy has the following rules:
+
+.. code:: python
+
+    # Repository Permissions
+    # ----------------------
+
+    # Repository members can read and list issues for the repo
+    allow(user: User, action: String, repo: Repository) if
+        repo.is_member(user) and
+        action in ["READ", "LIST_ISSUES"];
+
+    # Repository admins can list roles and delete the repo
+    allow(user: User, action: String, repo: Repository) if
+        repo.is_admin(user) and
+        action in ["LIST_ROLES", "DELETE"];
+
+    # Members of the parent organization can create new repos
+    allow(user: User, "CREATE", repo: Repository) if
+        repo.organization.is_member(user);
+
+The users Mike and Sully have the following attributes:
+
+-  Mike and Sully are both members of the parent organization (Monsters
+   Inc.), so they can both create repositories in the organization
+-  Mike is the admin of the "Paperwork" repository, so he can list roles
+   and delete the repo, in addition to reading and listing issues
+-  Sully is a member of the "Paperwork" repository, so he can only read
+   the repo and list issues
+
+Based on these user attributes and our policy, we can see why Mike is
+allowed to take more actions on the repository than Sully.
+
+With this relatively straightforward policy, it's easy to trace where
+the users' allowed actions come from. But ``get_allowed_actions()`` can
+be especially powerful with more complicated policies. For example, if
+we used oso's `SQLAlchemy Roles library
+features <https://docs.oso.dev/getting-started/roles/sqlalchemy/basic_roles.html>`__,
```

Comment:
Is the double underscore here intentional?

Reply:
No I used some auto convert from markdown to rst thing, came out weird. Will fix